extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubePlaylistExtractor.java

```diff
@@ -64,17 +70,40 @@ public YoutubePlaylistExtractor(final StreamingService service,
     @Override
     public void onFetchPage(@Nonnull final Downloader downloader) throws IOException,
             ExtractionException {
+        final String playlistId = getId();
+
         final Localization localization = getExtractorLocalization();
         final byte[] body = JsonWriter.string(prepareDesktopJsonBuilder(localization,
                         getExtractorContentCountry())
-                        .value("browseId", "VL" + getId())
+                        .value("browseId", "VL" + playlistId)
                         .value("params", "wgYCCAA%3D") // Show unavailable videos
                         .done())
                 .getBytes(StandardCharsets.UTF_8);
 
-        browseResponse = getJsonPostResponse("browse", body, localization);
-        YoutubeParsingHelper.defaultAlertsCheck(browseResponse);
+        browseMetadataResponse = getJsonPostResponse("browse",
+                List.of("$fields=" + SIDEBAR + "," + HEADER + "," + MICROFORMAT + ",alerts"),
+                body,
+                localization);
+
+        YoutubeParsingHelper.defaultAlertsCheck(browseMetadataResponse);
         isNewPlaylistInterface = checkIfResponseIsNewPlaylistInterface();
+
+        final PlaylistContinuation playlistContinuation = PlaylistContinuation.newBuilder()
+                .setParameters(ContinuationParams.newBuilder()
+                        .setBrowseId("VL" + playlistId)
+                        .setPlaylistId(playlistId)
+                        .setContinuationProperties(PLAYLIST_CONTINUATION_PROPERTIES_BASE64)
+                        .build())
+                .build();
+
+        initialBrowseContinuationResponse = getJsonPostResponse("browse",
```

Comment:
I guess there is no way to avoid doing two network requests, right? Doesn't initialBrowseContinuationResponse also contain the metadata?

Reply:
Only legacy data is present (which is what we use almost every time, as on most playlists YouTube switched to view models, we do not support this data). I can also filter the continuations to only return videos, but this add more detection possibilities for YouTube.

Reply:
I do not understand, so you mean we could only do one request? Then I'd suggest doing only one call even if it means slightly degraded data (although it depends to which degree it's degraded)

Reply:
> I suspect YouTube will remove it at some point and I think the fact it is returned in all continuations is more a bug than something else.

Okok then let's not use and be future proof

> I don't think we should be worried on this adding more detection possibilities for YouTube on metadata endpoints

True